### ISSUE TITLE
dstat for raspberry pi constantly crashing

### ISSUE THREAD

**Issue by Inkertus** [JustinAzoff/dstat-raspberry-pi#1]
after leaving the program running in terminal for a while, with regularity, it crashes out on me. Attached is a screenshot of the traceback.

![constant crashing](https://user-images.githubusercontent.com/39816686/141600942-966c0027-535c-40dd-bf9a-8d0f2cdbaf3e.png)

**Comment by JustinAzoff** [comment]
I don't think that error is caused by this script.  Unfortunately the error doesn't say what 'name' is when it crashes.  All this script does is read the value in `/sys/devices/virtual/thermal/thermal_zone0/temp` and divide it by 1000, and that shouldn't result in a value of 'infinity'.

maybe try running something like

    dstat -whatever-flags-you-are-using ; cat /sys/devices/virtual/thermal/thermal_zone0/temp

to see what that file contains when dstat crashes.

Could also try running a 2nd instance of dstat without the raspberry pi plugin to see if that one crashes as well.
